src/vaadin-date-time-picker.html

```diff
@@ -47,8 +93,204 @@
 
         static get properties() {
           return {
+            /**
+             * The value for this element.
+             *
+             * Supported date time format is based on ISO 8601 (without a time zone designator):
+             * - Minute precision `"YYYY-MM-DDThh:mm"`
+             * - Second precision `"YYYY-MM-DDThh:mm:ss"`
+             * - Millisecond precision `"YYYY-MM-DDThh:mm:ss.fff"` (default)
```

Comment:
Currently the value format depends a bit on how you set it. I think there was some case depending on if you set it via JS or attribute or user selection that it uses the millisecond precision now. If you set it via JS it tried to keep the format you set it in. This is a bit fuzzy atm since `date-time-picker` doesn't yet support the `step` property propagation to `time-picker` and doesn't really know what the value precision should be.

This should become better when `step` support is added (this would be the next step in #3).

Reply:
Ok, I guess then it's good for now.